editor/modes/visual-editor/style.scss

```diff
@@ -537,3 +537,22 @@ $sticky-bottom-offset: 20px;
 .visual-editor__invalid-block-warning-buttons .components-button {
 	margin-bottom: 5px;
 }
+
+.blocks-visual-editor__block-html textarea {
```

Comment:
Minor: In aiming to avoid the tag name selector, and since we don't apply any styles to `blocks-visual-editor__block-html`, maybe we skip applying this class name to the root element and instead target the textarea with a classname `.blocks-visual-editor__block-html-textarea`?